svgdigitizer/electrochemistry/cv.py

```diff
@@ -735,10 +736,68 @@ def comment(self):
 
         return comments[0].value
 
+    @property
+    def simultaneous_measurements(self):
+        r"""
+        Returns the names of additional measurments which are plotted
+        along with the digitized data in the same figure or subplot.
+
+        The names are read from a ``<text>`` in the SVG file such as
+        ``<text>simultaneous measurements: SXRD</text>``.
```

Comment:
Do these double backticks work here?

Reply:
Single backticks render the text in between *italic*. double backticks make the text look like `code`.